pyqtgraph/graphicsItems/LegendItem.py

```diff
@@ -339,11 +346,20 @@ def mouseDragEvent(self, ev):
             dpos = ev.pos() - ev.lastPos()
             self.autoAnchor(self.pos() + dpos)
 
+    def mouseDoubleClickEvent(self, ev):
+        self.sigDoubleClicked.emit(self, ev)
```

Comment:
We should probably pass the event to the superclass.

```python
super().mouseDoubleClickEvent(ev)
self.sigDoubleClicked.emit(self, ev)
```

Reply:
In my case I wanted the app get a notification, I'm not making a new class or subclass.  If you try out the Legend.py example I updated you'll see what I mean.  I use the double click signal to pop up a dialog box to change the legend font size.  Of course other style changes could be made available as well.